desc/integrals/bounce_utils.py

```diff
@@ -339,24 +360,24 @@ def bounce_quadrature(
     Returns
     -------
     result : jnp.ndarray
-        Shape (P, S, num_well).
-        Quadrature for every pitch.
+        Shape (P, M, L, num_well).
         First axis enumerates pitch values. Second axis enumerates the field lines.
-        Last axis enumerates the bounce integrals.
+        Third axis enumerates the flux surfaces. Last axis enumerates the bounce
+        integrals.
 
     """
-    errorif(z1.ndim != 3 or z1.shape != z2.shape)
     errorif(x.ndim != 1 or x.shape != w.shape)
-    pitch = jnp.atleast_2d(pitch)
+    errorif(z1.ndim != 4 or z1.shape != z2.shape)
+    errorif(pitch_inv.ndim != 3)
     if not isinstance(f, (list, tuple)):
-        f = [f]
+        f = list(f)
```

Comment:
if a single array is passed in wont this convert it to a list of float, not a length 1 list of array?